2022/24xxx/CVE-2022-24897.json

```diff
@@ -1,18 +1,101 @@
 {
```

Comment:
' https://jira.xwiki.org/browse/XWIKI-5168' does not match '^(ftp|http)s?://\\S+$'